tokio/src/runtime/mod.rs

```diff
@@ -146,6 +146,8 @@ mod current_thread;
 
 #[cfg(feature = "blocking")]
 mod enter;
+#[cfg(feature = "blocking")]
+pub(crate) use self::enter::{enter, try_enter};
```

Comment:
I think the `try_enter` reexport needs additional feature flags? https://dev.azure.com/tokio-rs/Tokio/_build/results?buildId=3183&view=logs&jobId=f52841cf-0b12-5937-32d0-289a461f9d93&taskId=bbd29543-dc08-5365-06a4-48c5ab3c16f2&lineStart=53&lineEnd=54&colStart=1&colEnd=1

Reply:
Yep... we need to figure out how to cleanup the feature flag mess.